packages/cli/src/gemini.tsx

```diff
@@ -86,7 +87,30 @@ async function relaunchWithAdditionalArgs(additionalArgs: string[]) {
 }
 import { runAcpPeer } from './acp/acpPeer.js';
 
+export function setupUnhandledRejectionHandler() {
+  let unhandledRejectionOccurred = false;
+  process.on('unhandledRejection', (reason, _promise) => {
+    const errorMessage = `=========================================
+CRITICAL: Unhandled Promise Rejection!
+=========================================
+This is an unexpected error. Please file a bug report using the /bug tool.
```

Comment:
Can you put this line at the very top of this error so it's the first thing people read (especially the /bug part)

Reply:
done